### PR TITLE
Viz 88 buttons

### PR DESCRIPTION
**Pull request by boyoonc**
additions to add more buttons to signal.

### CODE REVIEW

File: src/sass/bootstrap/modified-buttons.scss [hunk 1]
```diff
@@ -28,4 +32,95 @@
...
   }
+  &.disabled{
+    background-color: $mm-gray-1;
+    color: $mm-white;
+  }
+}
+
+.mmui-link-secondary-button{
+  //height: 100%;
+  display: inline-block; //to enable giving height
```
Comment (besseym) [inline src/sass/bootstrap/modified-buttons.scss]: We're not using any button styles from Bootstrap? No extension?

Comment (boyoonc) [reply src/sass/bootstrap/modified-buttons.scss]: They say in the documentation "The .btn classes are designed to be used with the <button> element. However, you can also use these classes on <a> or <input> elements (though some browsers may apply a slightly different rendering). (https://getbootstrap.com/docs/4.0/components/buttons/#button-tags) So in a previous pull request I had commented not to extend the .btn class for link buttons. 

I just double checked that extending the class doesn't seem to make a difference, so I can add it into all the link button classes if that is better.

Comment (besseym) [reply src/sass/bootstrap/modified-buttons.scss]: Not sure if its better to extend or not. Just wanted to make sure. I'll trust your judgement.

Comment (boyoonc) [reply src/sass/bootstrap/modified-buttons.scss]: Ok, I'll keep an eye on it to see if the bootstrap4 release makes any updates.

**Pull request merged**